crates/criticalup-core/src/download_server_client.rs

```diff
@@ -12,6 +12,12 @@ use reqwest::blocking::{Client, RequestBuilder, Response};
 use reqwest::header::{HeaderValue, AUTHORIZATION};
 use reqwest::StatusCode;
 use serde::Deserialize;
+use std::thread;
+use std::time::Duration;
+
+const CLIENT_TIMEOUT_DEFAULT: u64 = 5;
```

Comment:
```suggestion
const CLIENT_TIMEOUT_SECONDS u64 = 5;
```

No need to say `DEFAULT` since it is not configurable.